tests/conjur-client-files/output/hftoken.txt

```diff
@@ -1 +1 @@
-17jpdts3mtzks71geh5ah3w477b93kn000b1erf6cp36hjmam2tfmx30
+2t81g6xbgv5psazr4cv9rg7zg1kayx1g2w9qzjh133wf0q1339z9s
```

Comment:
Should this file be checked in if it's being generated?

Reply:
My mistake , will be deleted